### DISCUSSION TITLE
usługi Azure w Chinach

### DISCUSSION THREAD

**Discussion by pawelserwan** [AzureCommunityPL/group#197] [Pomoc] [answered]
Cześć,

Pytanie odnośnie usług Azure w datacenter w Chinach. Przeczytałem informacje na stronach MS. Ale chciałem zasięgnąć informacji na grupie. Czy ktoś z Was hostował coś w datacenter w Chinach? Generalnie potrzebujemy postawić tam VMki (3-4) i podpiać je tunelem VPN do reszty swiata. Jak to wyglada od strony transferu i bezpieczenstwa danych?

Dzięki,
Paweł

**Comment by wi5nia** [comment, marked as answer]
Generalnie dziala dobrze, tylko strasznie duzo papierkologii jest i jak nie masz kogos on-site w China to nie wroze Ci sukcesu zrobienia tego bo musisz tez miec zalozony ICP number, a do tego wymagana jest firma zarejestrowana w mainland China.

[https://docs.microsoft.com/en-us/azure/china/overview-connectivity-and-interoperability](https://docs.microsoft.com/en-us/azure/china/overview-connectivity-and-interoperability)

**Reply by pawelserwan** [reply]
dzięki @wi5nia ! Mamy tam zerejestrowaną spółkę i dużo ludzi on-site. Więc jakoś papierkologię przejdziemy mam nadzieję. Bardziej zastanawia mnie bezpieczeństwo i dostęp do danych w Azure China. Czy stawiania RODC jako VMki w Azure China jest jakimś zagrożeniem od strony security czy nie. I czy też postawienie VPN tunnel z Azure China do srodowiska on-prem w Europie może być jakimś problemem od strony nietechnicznej? Przepraszam jesli to "oczywiste" pytania ale do tej pory nie pracowałem z Azure w tamtej częsci swiata :)

**Reply by wi5nia** [reply]
Chiny to wogole ciezko temat... MS niby zapewnia o bezpieczenstwie, proywatnosci etc. natomiast musisz pamietac, ze Azure China nie jest operowany przez MSFT w Chinach tylko firme Chinska... 

Znam security teamy, ktore sie nie zgodzily na VPN czy ER pomiedzy Chinami a infrastruktura lokalna firmy.

Wiec to wszystko sie sprowadza do zaufania tego co pisze MS w sprawie bezpieczenstwa. Jak temu ufasz i wierzysz - go for it... jak nie... no to juz Twoja decyzja :)

Ja uzywam od prawie roku juz Azure China regularnie i poki co poza formalnosciami to zadnych problemow.

**Reply by pawelserwan** [reply]
dzięki @wi5nia . To już teraz decyzja po stronie security. Tak wiem że to oddzielna infrastruktura od całego Azure i lokalny hosting provider. Teoretycznie zawsze jak robisz jakikolwiek ruch do Chin to temat bezpieczeństwa jest aktualny.
Dzięki i miłego weekendu
